nix-tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -xueo pipefail
+
+which kmir
+kmir --help
+
+kmir parse --output pretty tests/arithm-simple.mir > tests/nix-arithm-simple.parse.out 
+git --no-pager diff tests/nix-arithm-simple.parse.out tests/arithm-simple.parse.out
+
+kmir run   --output pretty tests/arithm-simple.mir > tests/nix-arithm-simple.run.out 
+# Below fails, </generatedTop> does not get processed in poetry shell env
+# Uncomment and run `nix build --extra-experimental-features 'nix-command flakes' --print-build-logs .#kmir-test`
+#git --no-pager diff tests/nix-arithm-simple.run.out   tests/arithm-simple.run.out
```

Comment:
This error is noted in #200

Reply:
I would prefer hold off PR #199 until we figure out the reason of #200. 
It does not make sense to me if the two outputs (BTW, good work on comparing the outputs.) are inconsistent. 
Shall we consult the K frontend team or maybe pyk team to figure out why?

Reply:
Okay no problem, we can hold off. Let me have a bit of a look at it first. If I can't figure it out after some time I will reach out to those teams.